MadLibsUpdated.py

```diff
@@ -0,0 +1,127 @@
+    input9 = input("Input a verb(ending in ing):")
+    input10 = input("Input an adverb(ending in ly):")
+    input11 = input("Type a number:")
+    input12 = input("Measure of time:")
+    input13 = input("Input another color:")
+    input14 = input("Input a type of animal again:")
+    input15 = input("Type a number once again:")
+    input16 = input("Enter a silly word:")
+    input17 = input("Enter a noun last time:")
+
+    print(f"""
+        This weekend I am going camping with {input1}. 
+        I packed my lantern, sleeping bag, and {input2}. I am so {input3} to {input4} in a tent. 
+        I am {input5} we might see a(n) {input6}, I hear they’re kind of dangerous. 
+        While we’re camping, we are going to hike, fish, and {input7}. 
+        I have heard that the {input8} lake is great for {input9}. 
+        Then we will {input10} hike through the forest for {input11} {input12}. 
+        If I see a {input13} {input14} while hiking, I am going to bring it home as a pet! 
+        At night we will tell {input15} {input16} stories and roast {input17} around the campfire!!
+    """)
+
+
+def third_temp():
+    input1 = input("input a  proper noun(Person's name):")
+    input2 = input("Input an adjective:")
+    input3 = input("input a color:")
+    input4 = input("Input a type of animal:")
+    input5 = input("Input a place:")
+    input6 = input("Input an adjective again:")
+    input7 = input("Magical Creature(Plural):")
+    input8 = input("Input the third adjective:")
+    input9 = input("Magical creatue(Plural):")
+    input10 = input("Input a room in a house:")
+    input11 = input("Input another noun:")
+    input12 = input("Input a noun one more time:")
+    input13 = input("Input a noun in a plural form:")
+    input14 = input("Enter the fourth adjective:")
+    input15 = input("Input another noun(Plural):")
+    input16 = input("Type a number:")
+    input17 = input("Measure of time:")
+    input18 = input("Enter a verb(ending in ing):")
+    input19 = input("Type the last adjective:")
+    input20 = input("And finally, enter a noun:")
+  
+    print(f"""
+        Dear {input1}, I am writing to you from a {input2} castle in an enchanted forest. 
+        I found myself here one day after going for a ride on a {input3} {input4} in {input5}. 
+        There are {input6} {input7} and {input8} {input9} here! 
+        In the {input10} there is a pool full of {input11}. 
+        I fall asleep each night on a {input12} of {input13} and dream of {input14} {input15}. 
+        It feels as though I have lived here for {input16} {input17}. 
+        I hope one day you can visit, although the only way to get here now is {input18} on a {input19} {input20}!!
+    """)
+
+while keep_asking:
+    temp_input = input("Choose one out of the three templates(Type Y for random choice):")
+    
+
+    if (temp_input == 'y') or (temp_input == 'yes') or (temp_input == 'Y') or (temp_input == 'Yes') or (temp_input == 'YES'):
```

Comment:
if temp_input.lower() in ["y", "yes"]:
I suggest this code. This shorter and understandable․

Reply:
Thanks. But it was stated that we can't use built-in functions in this project. I don't have the right to use .lower() function.

Reply:
I think that if string is specified in requirements, then string modules can be used.

Reply:
Sorry, but I don't share the same opinion

Reply:
Okay